src/image-base.ks

```diff
@@ -18,7 +18,7 @@ firewall --disabled
 # rw and $ignition_firstboot are used by https://github.com/coreos/ignition-dracut/
 # Console settings are so we see output everywhere
 # %%KARGS%% is for distro-specific arguments
-bootloader --timeout=1 --append="console=ttyS0,115200n8 console=tty0 rootflags=defaults,prjquota rw %%KARGS%% $ignition_firstboot"
+bootloader --timeout=1 --append="console=%%TERM%%,115200n8 console=tty0 rootflags=defaults,prjquota rw %%KARGS%% $ignition_firstboot"
```

Comment:
Minor/optional: WDYT about just naming this `VM_TERMINAL` too? Makes it obvious what'll go there. And `TERM` has a different meaning.

Reply:
Good point. IMHO refactoring the ```VM_TERMINAL``` to ```TERMINAL``` or a like(```DEFAULT_TERMINAL```???), makes more sense.

Reply:
WDYT?

Reply:
Yeah, `DEFAULT_TERMINAL` SGTM!

Reply:
Done.